k8s-dev/cstor-pool-mgmt/replica-resource-1.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: openebs.io/v1alpha1
+kind: CStorVolumeReplica
+metadata:
+  name: pvc-ee171da3-07d5-11e8-a5be-42010a8001be-cstor-rep-9440ac
+  labels:
+#cstorpool uid will be used for reference and matching pool name
+   cstorpool.openebs.io/uid : 6eb3a8a5-707e-11e8-8a9f-54e1ad4a9dd4
+
+#this pool name is added as additional/alias name with zpool create command.
+   cstorpool.openebs.io/name: pool1-abc
+
+#pvc name is written in cvr label for reference.
+   cstorvolumereplica.openebs.io/pvc-name: demo-vol
+
+#cstorvolume uid will be used in zfs create command as zfs volname.
+   cstorvolume.openebs.io/uid: 6a55-11e8-a958-54e1ad4a9dd4-c3eaf637
+
+#finalizer helps in handling cvr deletion.
+  finalizers: ["cstorvolumereplica.openebs.io/finalizer"]
```

Comment:
Can we set up k8s rules that auto injects finalizers?

Reply:
As discussed, since CAS template creates this deployment yaml and user is not involved in applying the yaml file, this is not required to be handled by controller.